apps/admin/app/(dashboard)/(modules)/authentication/strategies/page.tsx

```diff
@@ -47,7 +47,7 @@ export default async function Strategies({
       </div>
       <div className={'grid grid-cols-6 gap-3 py-5'}>
         {enabledStrategies
-          .map((strategy) => <StrategyCard
+          .map((strategy, index) => <StrategyCard key={index}
```

Comment:
use the name of the strategy or another identifier. Using index on this one is incorrect.